pxtsim/libgeneric.ts

```diff
@@ -20,17 +20,14 @@ namespace pxsim {
         }
 
         toDebugString(): string {
```

Comment:
this will need rewritting. ``toDebugString`` should also receive the remaining number of characters as an argument -- and shortcut the recursion when the max is reached.
Can be done in future PR.